templates/homepage/grid_of_the_week_partial.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+<div class="text-center categories col-sm-4">
+    <h2>{% trans "Featured Grid" %}</h2>
+    {% if gotw %}
+        <h4><a href="{{ gotw.get_absolute_url }}">{{ gotw.title }}</a></h4>
+        <p>{{ gotw.description }}</p>
+        <p>
+            <strong>{% trans "Packages" %}:</strong> {{ gotw.gridpackage_set.count }}
+            <strong>{% trans "Features" %}:</strong> {{ gotw.feature_set.count }}
```

Comment:
We can pre-calculate thees using aggregation.

Reply:
@saadmk11 would you do that in the view or in the template? 

I can kind of see how I might start to get that idea set up in the view, but then my idea would kind of fall apart after that and get more complicated than it would probably need to be 😩

Any ideas on how to implement would be welcome 🙏

Reply:
We can add something like this
```python
potw = (
    Package.objects.filter(
        dpotw__start_date__lte=my_today, dpotw__end_date__gte=my_today
    )
    .annotate(grid_package_count=Count("gridpackage"))
    .only(
        "pk",
        "slug",
        "title",
        "repo_description",
    )
    .latest(
        "dpotw__end_date",
    )
)
```

Reply:
@saadmk11 thanks! I was trying to overly complicate it but this will get me in the right spot 😁